example/pubspec.yaml

```diff
@@ -24,12 +24,11 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
 
-  image_picker: ^0.4.10
-  
-  image: ^2.1.4
+  image_picker: ^0.6.7+12
 
-  tflite:
-    path: ../
+  image: ^2.1.18
+
+  tflite: ^1.1.1
```

Comment:
I think this should be set back to the following so that it uses the version from the cloned repository. This will prevent issues where the released version is different to the version depended upon by the example:

```yaml
  tflite:
    path: ../
```